src/app/auth/auth.guard.ts

```diff
@@ -18,6 +18,10 @@ export class AuthGuard implements CanActivate {
         return false;
       }
 
+    if (route.data.admin !== this.authService.isAdmin()) {
```

Comment:
No entiendo que hace esta línea. Si `this.authService.isAdmin()` te da un booleano no parece una buena comparación. ¿Puedes explicar?

Reply:
route.data.admin lo puse como un booleano que me dice que solo el admin puede acceder a ella, entonces las rutas que tienen el admin en false solo pueden ser accedidas por personas que no son admin, y las que tienen en true solo pueden ser accedidas por el admin